editor.planx.uk/src/@planx/components/NextSteps/model.ts

```diff
@@ -22,3 +26,17 @@ export const parseNextSteps = (
 });
 
 const DEFAULT_TITLE = "What would you like to do next?";
+
+export const validationSchema: SchemaOf<NextSteps> = baseNodeDataValidationSchema.concat(
+  object({
+    title: string().required(),
+    description: richText(),
+    steps: array(
+      object({
+        title: string().required(),
+        description: string().required(),
+        url: string().url(),
+      })
+    ).min(1),
```

Comment:
I've not yet tackled trying to pass these errors down into the `ListManager` (which is why I've also skipped `FileUpload` and `FileUploadAndLabel`)